### DISCUSSION TITLE
globeview custom basemap image

### DISCUSSION THREAD

**Discussion by nphias** [visgl/deck.gl#7356] [Q&A] [answered]
There are a few webgl globe projects out there. 
Most let you choose the image to wrap around the globe.. like nasa's blue earth marble.. 

I would really appreciate some guidance on how to do this..  is it possible to do with the globeview ui in deck.gl...  i would really like to stay with this framework .

issue on stackoverflow:
https://stackoverflow.com/questions/73950302/deck-gl-custom-basemap-with-globe-ui-using-blue-marble

**Comment by Pessimistress** [comment, marked as answer]
```js
new BitmapLayer({
      id: 'base-map-raster',
      image: WORLD_MAP,
      bounds: [-180, -90, 180, 90]
})
```

**Comment by tmcw** [comment]
On a similar note, when I try to use Deck.gl with a Mapbox GL JS map that uses their 3D globe projection, the Deck overlays only appear once I've zoomed far enough in that Mapbox has warped the map back to Mercator projection. Is this the expected behavior, or is there an issue tracking the ability to use Deck with Mapbox's globe view?

**Reply by Pessimistress** [reply]
Mapbox does not currently support third party renderers in their non-Mercator projections. The projection parameters are not exposed in any way so we cannot synchronize with their map.
